### PR TITLE
add token

### PR DESCRIPTION
**Pull request by loko8888**

### CODE REVIEW

File: cosmos/injective.json [hunk 1]
```diff
@@ -70,6 +70,12 @@
       "coinDecimals": 6,
       "coinMinimalDenom": "factory/inj127l5a2wmkyvucxdlupqyac3y0v6wqfhq03ka64/qunt",
       "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/injective/factory/inj127l5a2wmkyvucxdlupqyac3y0v6wqfhq03ka64/QUNT.png"
+      },
+    {
+      "coinDenom": "NONJA",
+      "coinDecimals": 18,
+      "coinMinimalDenom": "inj1fu5u29slsg2xtsj7v5la22vl4mr4ywl7wlqeck",
```
Comment (HeesungB) [inline cosmos/injective.json:77]: @loko8888 Could you check your `coinMinimalDenom`?
I think you need `factory/` in front of it.

Comment (loko8888) [reply cosmos/injective.json:77]: Yes was wondering too but it's a CW20 token, so there is no factory.

Comment (loko8888) [reply cosmos/injective.json]: I added their new denom factory/inj14ejqjyq8um4p3xfqj74yld5waqljf88f9eneuk/inj1fu5u29slsg2xtsj7v5la22vl4mr4ywl7wlqeck should be all good now.

**Comment by HeesungB** [comment]
@loko8888 Would you add CW token on this repository?
Cw tokens are managed here.

https://github.com/chainapsis/keplr-contract-registry

**Pull request closed without merging**